data/config/features/MLFeatures.json

```diff
@@ -1,27 +1 @@
-{
-  "MarkLogicFeatures": {
-    "Data Integration": [
-      "Entity Services",
-      "Data Movement SDK",
-      "Template Driven Extraction",
-      "SQL Enhancements",
-      "Optic API"
-    ],
-    "Security": [
-      "Encryption",
-      "Redaction",
-      "Element-level Security"
-    ],
-    "Manageability": [
-      "Ops Director",
-      "New Relic Plugin"
-    ],
-    "Additional Enhancements": [
-      "Geospatial",
-      "Search",
-      "Enhanced Tiered Storage",
-      "JavaScript",
-      "Query Console"
-    ]
-  }
-}
+{"MarkLogicFeatures":{"Data Integration":["Entity Services", "Data Movement SDK", "Template Driven Extraction", "SQL Enhancements", "Optic API"], "Security":["Encryption", "Redaction", "Element-level Security"], "Manageability":["Ops Director", "New Relic Plugin"], "Additional Enhancements":["Geospatial", "Search", "Enhanced Tiered Storage", "JavaScript", "Query Console"]}}
```

Comment:
because this is all on one line, it's really difficult to see what has changed (if anything). Please reformat to match the way it was before.